app/javascript/src/pages/more/index.js

```diff
@@ -6,6 +6,7 @@ import Config from '../../config'
 import jsontable from '../../utils/jsonApi'
 import { titlebarNormal } from '../../utils/tablebar'
 import { trackDapplist } from '../../utils/sensors'
+import {currentLocale} from "../../i18n"
```

Comment:
Keep a space after `{` and before `}` as line 8.

Reply:
Updated now